modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -169,6 +169,11 @@ include::example$configuration/user/user_auth_ldap/group_filter.ldif[]
 This is an indicator that tells you approximately how many users will be listed in ownCloud. 
 The number updates automatically after any changes.
 
+[IMPORTANT]
+====
+No matter what search filter is given the app always add the value of the `User Display Name Field` (default `(displayName=*)`) to the filter so ensure that your users have this field otherwise the count will be 0.
```

Comment:
```suggestion
No matter what search filter is given the app always adds the value of the `User Display Name Field` (default `(displayName=*)`) to the filter so ensure that your users have this field otherwise the count will be 0.
```